samples/hello_invalid.k

```diff
@@ -0,0 +1,6 @@
+name = "kcl"
```

Comment:
Do not add illegal examples to examples.

Reply:
fixed.